packages/form-builder/src/blocks/fields/amount/Edit.tsx

```diff
@@ -1,51 +1,154 @@
+            })}
+        </LevelGrid>
+    );
+
+    const CustomAmount = () => (
+        <CurrencyControl value={setPrice} label={__('Custom amount', 'give')} hideLabelFromVision />
+    );
 
-    const FixedPriceMessage = () => {
-        const amount = formatCurrencyAmount(setPrice);
+    const FixedPriceMessage = () => (
+        <Notice>
+            {createInterpolateElement(__('This donation is set to <amount/> for this form.', 'give'), {
+                amount: <strong>{amountFormatted}</strong>,
+            })}
+        </Notice>
+    );
 
-        return createInterpolateElement(__('This donation is set to <amount/> for this form.', 'give'), {
-            amount: <strong>{amount}</strong>,
-        });
+    const RecurringPeriod = ({count}) => {
+        const interval = count ?? recurringBillingInterval;
+
+        const singular = !isRecurringDonor
+            ? periodLookup[recurringBillingPeriod].singular
+            : periodLookup[recurringOptInDefaultBillingPeriod ?? 'month'].singular;
+
+        const plural = !isRecurringDonor
+            ? periodLookup[recurringBillingPeriod].plural
+            : periodLookup[recurringOptInDefaultBillingPeriod ?? 'month'].plural;
+
+        return (
+            <strong>
+                {1 === interval && <>{singular}</>}
+                {1 !== interval && (
+                    <>
+                        {interval} {plural}
+                    </>
+                )}
+            </strong>
+        );
+    };
+
+    const RecurringMessage = () => {
+
+        const interval = parseInt(recurringBillingInterval);
+
+        const periodPlaceholder = <strong>{'[' + _n('period', 'periods', interval, 'give') + ']'}</strong>;
+
+        const count = parseInt(recurringLengthOfTime);
+        const countElement = <strong>{count}</strong>;
+
+        const translatableString = !count
+            ? __('This donation occurs every <period />.', 'give')
+            : __('This donation occurs every <period /> for <count /> <strong>payments</strong>.', 'give');
+
+        return (
+            <Notice>
+                { 'one-time' === recurringOptInDefaultBillingPeriod
+                    ? __('You have chosen to make this donation one-time.', 'give')
+                    : createInterpolateElement(translatableString, {
+                        period: <RecurringPeriod count={interval} />,
+                        count: countElement,
+                        strong: <strong />, // Required to interpolate the strong tag around the "payments" text.
+                    })
+                }
+            </Notice>
+        );
+    };
+
+    const FixedRecurringMessage = () => (
+        <Notice>
+            {createInterpolateElement(__('This donation is <amount/> every <period/>. ', 'give'), {
+                amount: <strong>{amountFormatted}</strong>,
+                period: <RecurringPeriod count={parseInt(recurringBillingInterval)} />,
+            })}
+        </Notice>
```

Comment:
We need to avoid this from happening (should just be the single fixed message if one-time:

<img width="1102" alt="Screenshot 2023-03-09 at 11 22 38 AM" src="https://user-images.githubusercontent.com/10138447/224087195-10420d97-74dd-475c-846a-b2910857419d.png">